src/content/plugins/progress-plugin.md

```diff
@@ -39,26 +40,77 @@ new webpack.ProgressPlugin(handler);
 
 When providing an `object` to the `ProgressPlugin`, following properties are supported:
 
-- `activeModules` show's active modules count and one active module in progress message
```

Comment:
this list was a good summary of properties, if you wanted to make it more accessible you could aggregate defaults into this list. Spreading it out the page and copying over the same info just makes it different and not necessarily improve the documentation

Reply:
I followed this guide https://webpack.js.org/contribute/writers-guide/#configuration-defaults-and-types
All the options are listed in the example ahead
Do we really need to revert this changes?

Reply:
they are usually done per config property, since config property itself is object its fine to keep it as a list, what i meant is adding types and defaults (as you did) but to this list to keep it as a short summary (as info did not really expand much)

for example 
```diff
- - `activeModules` show's active modules ...
+ - `activeModules` (`boolean = false`): show's active modules ...
```

Like we did previously here: https://webpack.js.org/plugins/eval-source-map-dev-tool-plugin/#options